LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -47,19 +59,30 @@ public  void CreateAccounts(int accountCount)
                     }
                 }
             }
+
+            ReportSummary(actionName: action, itemCount: accountCount);
+            ReportFooter(actionName: action);
         }
 
         public void ActivateAccounts()
         {
+            string action = "Activate account";
+            ReportHeader(actionName: action);
+            int i = 0;
             foreach (var account  in pendingAccountList )
             {
                 account.Status = Status.Active; 
                 pendingAccountList.Remove(account);
+                ReportItem(itemName: account.ToString(), actionName: action, itemIndex: i);
             }
+            ReportSummary(actionName: action, itemCount: pendingAccountList);
+            ReportFooter(actionName: action);
         }
 
         public void CreateSubjects(int subjectCount)
         {
+            string action = "Subject generation";
+            ReportHeader(actionName: action);
             Console.WriteLine("******** Started creating subjects  ********\n");
```

Comment:
All writing to terminal should go through the report methods, remove Console.WriteLine().

Reply:
DONE